phidgets_msgs/srv/SetAnalogOutput.srv

```diff
@@ -0,0 +1,6 @@
+# Sets the state of a digital output. 
+
+uint16 index   # index of the output to be set
+float32 voltage
```

Comment:
The API call above and in the libphidget22 library take in a double, so I'll suggest switching this to a `float64` instead.